lib/improver/tests/utilities/cube_manipulation/test_concatenate_cubes.py

```diff
@@ -76,21 +83,14 @@ def test_cubelist_type_and_data(self):
         resulting data, if a CubeList containing non-identical cubes
         (different values for the time coordinate) is passed in as the input.
         """
-        cube1 = self.cube.copy()
-        cube2 = self.cube.copy()
-
         cube3 = self.cube.copy()
```

Comment:
Perhaps `cube3` could be renamed here, maybe just to be `cube`, given that `cube1` and `cube2` do not exist now.